src/documentation/dokumentacja_wstepna.md

```diff
@@ -35,3 +50,70 @@ Dla uproszczenia obliczeń, powyższe równanie można zapisać w sposób macier
 </p>
 
 Najczęściej wykorzystuje się do tego celu klasyczną metodę najmniejszych kwadratów i jej pochodne. Metoda ta jest najstarsza i najłatwiejsza do zastosowania, choć posiada wady (np. niewielką odporność na elementy odstające), które udało się usunąć we wspomnianych pozostałych metodach.
+
+### Agent globalny
+Agent globalny odpowiedzialny jest za przydzielanie zasobów chmury w taki sposób, aby
+zoptymalizować zużycie energii, a tym samym koszty utrzymania chmury. Optymalizacja odbywa się poprzez
+utrzymanie właściwego rozkładu maszyn wirtualnych na maszynach fizycznych oraz propagowanie zadań 
+od użytkowników na ich maszyny wirtualne.
+
+Agent globalny redukuje liczbę uruchomionych (a zatem pobierających energię) maszyn fizycznych poprzez 
+redukcję aktywnych maszyn wirtualnych i ich migracji. Dokonuje tego poprzez szybkie zwalnianie zasobów
```

Comment:
to zdanie chyba trochę brzmi tak jakby agent globalny redukował aktywne maszyny i liczbę migracji.

chyba lepiej brzmiałoby to jako "Agent globalny redukuje liczbę uruchomionych (a zatem pobierających energię) maszyn fizycznych poprzez migracje i zmniejszanie ilości aktywnych maszyn wirtualnych"

Reply:
Miałam na myśli redukcję liczby migracji